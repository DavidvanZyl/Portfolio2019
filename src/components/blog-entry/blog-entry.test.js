import React from "react"
import BlogEntry from "./blog-entry"

describe("BlogEntry", () => {
  test("renders correctly", () => {
    const wrapper = shallow(<BlogEntry item={{ testItem: "test" }} />)
    expect(wrapper).toMatchSnapshot()
  })
})
