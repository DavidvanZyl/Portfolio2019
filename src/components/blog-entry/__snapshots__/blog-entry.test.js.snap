// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogEntry renders correctly 1`] = `
<p
  className="blog"
/>
`;
